code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -303,7 +303,7 @@
 			enthrall_safe = 1
 			break
 	if(!C)
-		log_to_dd("something bad happened on enthralling a mob, attacker is [user] [user.key] \ref[user]")
+		log_runtime(EXCEPTION("something bad happened on enthralling a mob, attacker is [user] [user.key] \ref[user]"), user)
```

Comment:
Oh, wait, in-game. Would probably make sense to let the person input a ref to VV it